viscy/data/cell_classification.py

```diff
@@ -23,6 +23,7 @@ def __init__(
         transform: Callable | None,
         initial_yx_patch_size: tuple[int, int],
         return_indices: bool = False,
+        label_column: str = "infection_state",
```

Comment:
I guess we are being flexible here, but this opens the door for non standarization.

Reply:
Is developing the controlled vocabulary in scope for this PR?

Reply:
I made a new PR #300